core/src/main/kotlin/com/epam/drill/endpoints/AgentManager.kt

```diff
@@ -6,6 +6,7 @@ import com.epam.drill.agentmanager.*
 import com.epam.drill.api.*
 import com.epam.drill.common.*
 import com.epam.drill.endpoints.agent.*
+import com.epam.drill.endpoints.openapi.*
```

Comment:
Why this import?

Reply:
This is necessary to use SystemSettings, which are a function parameter.